### ISSUE TITLE
my masks_pre is negative number

### ISSUE THREAD

**Issue by gboy2019** [milesial/Pytorch-UNet#172]
![Uploading image.png…]()
tensor([[[[-0.0895, -0.2451, -0.3415,  ...,  0.5073,  0.0656, -0.0943],
          [ 0.2399,  0.4162,  0.6017,  ...,  1.0372, -0.2323,  0.2533],
          [-0.2687, -0.1455,  0.2024,  ...,  0.5458,  0.0667, -0.1614],
          ...,
          [-0.0476,  0.3979, -0.2003,  ..., -0.2475, -0.3053,  0.1256],
          [ 0.1851,  0.0701, -0.2488,  ..., -0.2048,  0.0101,  0.2742],
          [ 0.0211,  0.2133, -0.2180,  ..., -0.2522, -0.1804, -0.2069]]]],
       grad_fn=<MkldnnConvolutionBackward>) tensor([[[[0., 0., 0.,  ..., 0., 0., 0.],
          [0., 0., 0.,  ..., 0., 0., 0.],
          [0., 0., 0.,  ..., 0., 0., 0.],
          ...,
          [0., 0., 0.,  ..., 0., 0., 0.],

**Comment by gboy2019** [comment]
and loss  always -180 and -1e+3

**Comment by gboy2019** [comment]
![image](https://user-images.githubusercontent.com/52697246/81180126-2cc6ec80-8fdd-11ea-9333-b3537da3b957.png)

**Comment by gboy2019** [comment]
@milesial
Can you help me?Thank you very much.

**Comment by milesial** [comment]
Hi, for the negative loss problem look at this https://github.com/milesial/Pytorch-UNet/issues/173#issuecomment-626085316

**Comment by OmargarridoH** [comment]
get the same mask_predict, I am using 2 channels and 3 classes. on tensorboard I get the image right but the moment I run the predict I get these 
![Negative values predict array](https://user-images.githubusercontent.com/61413658/81486354-8e44c080-9254-11ea-943d-96f40d1d7f8e.PNG)

**Comment by milesial** [comment]
That strange, can you give me the mask before the `transforms.Compose` is applied?

**Comment by OmargarridoH** [comment]
that is the matrix after the softmax when classes are >1, so is after the transform.compose. so my softmax is giving my negative percentage.

**Comment by milesial** [comment]
If your softmax is giving you negative values that's not the problem on my repo, you should open a PyTorch issue.

**Comment by OmargarridoH** [comment]
thanks, I will work on that then .
